pyrit/prompt_converter/__init__.py

```diff
@@ -88,4 +88,5 @@
     "UnicodeSubstitutionConverter",
     "UrlConverter",
     "VariationConverter",
+    "PDFConverter"
```

Comment:
I think they list the converter name in alphabet, maybe you can move it from the end to somewhere between "O" and "Q"